pkg/repo/mysql/mysql.go

```diff
@@ -0,0 +1,34 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+type Conf struct {
+	Addr     string
```

Comment:
maybe best to separate address into two fields 'host' and 'port' for clarify.

Reply:
I guess, it will be better